js/main.js

```diff
@@ -646,6 +646,8 @@ require([
 		}
 
 		redrawMap();
+		// update map object in local storage
+		localStorage.setItem("mapObjects", JSON.stringify(mapObjects));
```

Comment:
> loading the map object into local storage on every change might cause performance issues if there are many drawings, but I doubt it would very be noticeable unless there was an unreasonable amount of them

Well this does sort of fire on every click, so it might be better to track if any changes have been made to mapObjects and only then save. We could also just set a flag here instead, then have a setInterval that fires every e.g. 5 seconds to make the actual save and avoid overly rapid saving, but all of this may actually just be more overhead than this currently is. 🤷 

I've tested some of the larger piles of coords from coordinate viewer I've got on hand and it doesn't seem to have any performance impact, but then again my localStorage is on an SSD so it may be worse on mobile with flash storage.